src/dev/run_i18n_integrate.ts

```diff
@@ -90,30 +91,32 @@ run(
 
     const srcPaths = Array().concat(path || ['./src', './packages']);
 
-    const list = new Listr([
+    const list = new Listr<ListrContext>([
       {
         title: 'Merging .i18nrc.json files',
         task: () => new Listr(mergeConfigs(includeConfig), { exitOnError: true }),
       },
       {
         title: 'Extracting Default Messages',
         task: ({ config }) =>
-          new Listr(extractDefaultMessages(config, srcPaths), { exitOnError: true }),
+          new Listr(extractDefaultMessages(config!, srcPaths), { exitOnError: true }),
       },
       {
         title: 'Integrating Locale File',
         task: async ({ messages, config }) => {
-          await integrateLocaleFiles(messages, {
-            sourceFileName: source,
-            targetFileName: target,
-            dryRun,
-            ignoreIncompatible,
-            ignoreUnused,
-            ignoreMissing,
-            ignoreMalformed,
-            config,
-            log,
-          });
+          if (config) {
```

Comment:
I like you pattern of throwing an error if config is missing instead of this

Reply:
Looks like you missed this change. The reason i like throwing an error is because it makes it clear why the task failed. The config should never be empty, but if it is, based on this change this task will complete silently, but the other tasks like `checkCompatibility` will throw an error. And since we expect config to be defined in this task, I want to know if it did not run.